helm/nginx-ingress-controller-app/templates/_helpers.tpl

```diff
@@ -61,6 +61,13 @@ LB Service name.
 {{ include "resource.default.name" . }}{{ .Values.controller.service.suffix }}
 {{- end -}}
 
+{{/*
+Internal LB Service name.
+*/}}
+{{- define "resource.controller-service-internal.name" -}}
+{{ include "resource.default.name" . }}-internal{{ .Values.controller.service.internal.suffix }}
```

Comment:
This could cause issues if a customer decides to use the suffix feature on an existing installation. helm will not remove the old service. But this is a general problem. Maybe a sentence in the changelog along the lines of "if you want to use this new feature, we recommend uninstalling and re-installing"

Reply:
Hm, well. This is the same for the existing `suffix` on the non-internal `LoadBalancer` service. But yeah, will add a note. Better safe than sorry! :)